### ISSUE TITLE
floating texture

### ISSUE THREAD

**Issue by Kramolnikk** [bgrabitmap/lazpaint#308]
Perhaps not a bug, but a feature
When using a "polygon or polyline" with a texture mapping, the texture is displaced during the construction of the shape, although this is not necessary, it is already repeated.

https://user-images.githubusercontent.com/42349207/134944652-73abfb8d-5060-4f28-82ff-aa12ff98f903.mp4

**Comment by circular17** [comment]
Yes, texture is aligned to the shape bounds. For example if you draw a rectangle with a small texture, it will align to the top left. This could be optional though
